content/12.privacy.md

```diff
@@ -16,17 +17,16 @@ For practitioners intending to distribute trained models, having an ethics regis
 Second, to help foster a conscious ethics-oriented mindset, researchers should consider expanding journal clubs to include scholarly and popular articles detailing real-world ethics issues relevant to their scientific fields.
 This will help researchers to think more holistically and judiciously about their work and its implications.
 Third, we encourage individual- and team-level participation in professional societies [@https://asbh.org] and other types of organizations [@https://web.archive.org/web/20210112231619/https://ocean.sagepub.com/blog/10-organizations-leading-the-way-in-ethical-ai] and events [@https://www.aies-conference.com/] related to the domains of AI and data ethics as well as bioethics.
-This will encourage a sense of community and intellectual engagement, keeping practitioners abreast of cutting-edge insights and emerging professional standards.
+This will encourage a sense of community and intellectual engagement, and will keep practitioners abreast of cutting-edge insights and emerging professional standards.
```

Comment:
```suggestion
This will encourage a sense of community and intellectual engagement and keep practitioners abreast of cutting-edge insights and emerging professional standards.
```

Reply:
I'm ok with keeping it then

Reply:
does it have to be converted to an independent clause though? Like

> This will encourage a sense of community and intellectual engagement, and *it* will keep ...